x/ibc/ante/ante.go

```diff
@@ -0,0 +1,47 @@
+package ante
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	client "github.com/cosmos/cosmos-sdk/x/ibc/02-client"
+	channel "github.com/cosmos/cosmos-sdk/x/ibc/04-channel"
+)
+
+// ProofVerificationDecorator handles messages that contains application specific packet types,
+// including MsgPacket, MsgAcknowledgement, MsgTimeout.
+// MsgUpdateClients are also handled here to perform atomic multimsg transaction
+type ProofVerificationDecorator struct {
+	clientKeeper  client.Keeper
+	channelKeeper channel.Keeper
+}
+
+// NewProofVerificationDecorator constructs new ProofverificationDecorator
+func NewProofVerificationDecorator(clientKeeper client.Keeper, channelKeeper channel.Keeper) ProofVerificationDecorator {
```

Comment:
I noticed this is being set neither at the app constructor nor on `x/auth/ante/`. We should add it or alternatively move `NewAnteHandler` from `/x/auth/ante` to `app.go`

Reply:
Isn't that in the `gaia` repository?